terraform/deployments/cluster-services/aws_efs_csi_driver.tf

```diff
@@ -0,0 +1,24 @@
+resource "helm_release" "efs_csi_driver" {
+  chart      = "aws-efs-csi-driver"
+  name       = "aws-efs-csi-driver"
+  namespace  = "kube-system"
+  repository = "https://kubernetes-sigs.github.io/aws-efs-csi-driver"
+  version    = "3.1.1" # TODO: Dependabot or equivalent so this doesn't get neglected.
+
+  values = [yamlencode({
+    controller = {
+      serviceAccount = {
+        create = true
+        name   = data.terraform_remote_state.cluster_infrastructure.outputs.aws_efs_csi_driver_controller_service_account_name
```

Comment:
```suggestion
        name   = data.tfe_outputs.cluster_infrastructure.nonsensitive_outputs.aws_efs_csi_driver_controller_service_account_name
```

Reply:
Ah I forgot to change this when I made the branch. All fixed.